functions/src/schedule-generator/speakers-sessions-map.js

```diff
@@ -0,0 +1,46 @@
+function sessionsSpeakersMap(sessionsRaw, speakersRaw) {
+    let sessions = {};
+    let speakers = {};
```

Comment:
can be defined as `const` as you don't change reference

Reply:
true